charts/bandstand-web-service/README.md

```diff
@@ -37,8 +37,12 @@
-| livenessProbe.path                  | string | `/`                                                           | Path to the endpoint used to determine whether a container is active                                                                                                                                                                                                                                       |
+| ingress.additionalPaths             | string | `.Release.Name`                                               | Additional path rules to apply to the ingress host, see [here](https://kubernetes.io/docs/concepts/services-networking/ingress/#the-ingress-resource) and [here](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#httpingresspath-v1-networking-k8s-io)                                |                                                                                                                                                                                                                                      |
+| kafka.bootstrapServersEnvVar                  | string | `KAFKA_BROKERS`                                                           | Comma separated list of Kafka brokers `hostname:port` to connect to for bootstrap.                                                                                                                                                                                                                                        |
+| kafka.consumerGroup                  | string |                                                            | Name of the consumer group used for checking the offset on the topic and processing the related lag.                                                                                                                                                                                                                                        |
+| kafka.topic                  | string |                                                            | Name of the topic on which processing the offset lag. lag.                                                                                                                                                                                                                                        |
+| kafka.tls                  | string | `disable`                                                           | To enable SSL auth for Kafka, set this to enable. If not set, TLS for Kafka is not used.                                                                                                                                                                                                                                        |
```

Comment:
The chart seems to defaulting this to 'enable'. I'm ok with it being enabled by default, but the docs should match (don't forget the headless-svc docs too)